### ISSUE TITLE
Wrong VRAM detection

### ISSUE THREAD

**Issue by Orbis1** [Pavelevich/llm-checker#76]
llm-checker | Hardware Detection
------------------------------------------
✔ Hardware detected!

```
=== Hardware Detection ===

Summary:
  2x AMD Ryzen 7 6800H with Radeon Graphics + gfx1035 + AMD Radeon Graphics + AMD + amdgcn-amd-amdhsa--gfx1035 + Rembrandt [Radeon 680M] + Radeon 680M (48GB VRAM) + AMD Ryzen 7 6800H with Radeon Graphics
  Tier: MEDIUM HIGH
  Max model size: 46GB
  Best backend: AMD ROCm
  Dedicated GPUs: gfx1035 + AMD + amdgcn-amd-amdhsa--gfx1035
  Integrated GPUs: 2x AMD Ryzen 7 6800H with Radeon Graphics + AMD Radeon Graphics + Rembrandt [Radeon 680M] + Radeon 680M

CPU:
  AMD Ryzen 7 6800H with Radeon Graphics
  Cores: 16 (8 physical)
  SIMD: AVX2
  [OK] AVX2

ROCM:
  ROCm: null
  Total VRAM: 48GB
  AMD Ryzen 7 6800H with Radeon Graphics: 8GB
  AMD Ryzen 7 6800H with Radeon Graphics: 8GB
  gfx1035: 8GB
  AMD Radeon Graphics: 8GB
  AMD: 8GB
  amdgcn-amd-amdhsa--gfx1035: 8GB

GENERIC:
  Source: systeminformation+lspci
  Total dedicated VRAM: 0GB
  Rembrandt [Radeon 680M]: 14GB (Integrated)
  Radeon 680M: shared/unknown (Integrated)

Fingerprint: rocm--ryzen-7-6800h-with-graphics-48gb-x6
```

But its total 32Gb RAM, and 4GB VRAM from iGPU Radeon 680M

**Comment by Pavelevich** [comment]
hi, is fixed, can u check if works, thx

**Comment by itsenyex** [comment]
Nice

**Comment by Orbis1** [comment]
> hi, is fixed, can u check if works, thx

```
npx llm-checker --version
3.5.12


npx llm-checker hw-detect

✔ Hardware detected!

=== Hardware Detection ===

Summary:
  2x AMD Ryzen 7 6800H with Radeon Graphics + gfx1030 + AMD Radeon Graphics + AMD + amdgcn-amd-amdhsa--gfx1030 + Rembrandt [Radeon 680M] + Radeon 680M (64GB VRAM) + AMD Ryzen 7 6800H with Radeon Graphics
  Tier: MEDIUM HIGH
  Max model size: 62GB
  Best backend: AMD ROCm
  Dedicated GPUs: gfx1030 + AMD + amdgcn-amd-amdhsa--gfx1030
  Integrated GPUs: 2x AMD Ryzen 7 6800H with Radeon Graphics + AMD Radeon Graphics + Rembrandt [Radeon 680M] + Radeon 680M

CPU:
  AMD Ryzen 7 6800H with Radeon Graphics
  Cores: 16 (8 physical)
  SIMD: AVX2
  [OK] AVX2

ROCM:
  ROCm: null
  Total VRAM: 64GB
  AMD Ryzen 7 6800H with Radeon Graphics: 8GB
  AMD Ryzen 7 6800H with Radeon Graphics: 8GB
  gfx1030: 16GB
  AMD Radeon Graphics: 8GB
  AMD: 8GB
  amdgcn-amd-amdhsa--gfx1030: 16GB

GENERIC:
  Source: systeminformation+lspci
  Total dedicated VRAM: 0GB
  Rembrandt [Radeon 680M]: 14GB (Integrated)
  Radeon 680M: shared/unknown (Integrated)

Fingerprint: rocm-gfx1030-64gb-x6
```
